crates/db/src/queries/erc20.rs

```diff
@@ -146,6 +146,57 @@ impl DbInner {
             .collect())
     }
 
+    pub async fn get_erc20_blacklist(
+        &self,
+        chain_id: u32,
+        address: Address,
+        include_blacklisted: bool,
```

Comment:
I dont' understand this function. from the name, sounds like we're asking for the list of blacklisted tokens

so what is the `include_blacklisted` arg doing?

Reply:
Yes it was redundant. I have updated!